src/time.rs

```diff
@@ -175,6 +245,243 @@ impl SubAssign<Duration> for GpsTime {
+    ) -> UtcParams {
+        let tot = tot.to_gps_time_t();
+        let t_lse = t_lse.to_gps_time_t();
+        UtcParams(c_bindings::utc_params_t {
+            a0,
+            a1,
+            a2,
+            tot,
+            t_lse,
+            dt_ls,
+            dt_lsf,
+        })
+    }
+
+    /// Modulo 1 sec offset from GPS to UTC [s]
+    pub fn get_a0(&self) -> f64 {
+        self.0.a0
+    }
+    /// Drift of time offset from GPS to UTC [s/s]
+    pub fn get_a1(&self) -> f64 {
+        self.0.a1
+    }
+    /// Drift rate correction from GPS to UTC [s/s]
+    pub fn get_a2(&self) -> f64 {
+        self.0.a2
+    }
+    /// Reference time of UTC parameters.
+    pub fn get_tot(&self) -> GpsTime {
+        GpsTime(self.0.tot)
+    }
+    /// Time of leap second event.
+    pub fn get_t_lse(&self) -> GpsTime {
+        GpsTime(self.0.t_lse)
+    }
+    /// Leap second delta from GPS to UTC before LS event [s]
+    pub fn get_dt_ls(&self) -> i8 {
+        self.0.dt_ls
+    }
+    /// Leap second delta from GPS to UTC after LS event [s]
+    pub fn get_dt_lsf(&self) -> i8 {
+        self.0.dt_lsf
+    }
+}
+
+impl Default for UtcParams {
+    fn default() -> Self {
+        unsafe { std::mem::zeroed::<UtcParams>() }
+    }
+}
+
+/// Structure representing UTC time
+///
+/// Note: This implementation does not aim to be able to represent arbitrary dates and times.
+/// It is only meant to represent dates and times over the period that GNSS systems have been
+/// around. Specifically it shouldn't be relied on for dates significantly before January 6th 1980,
+/// the start of GPS time.
+#[derive(Clone)]
+pub struct UtcTime(c_bindings::utc_tm);
+
+impl UtcTime {
+    pub(crate) fn mut_c_ptr(&mut self) -> *mut c_bindings::utc_tm {
+        &mut self.0
+    }
+
+    pub(crate) fn c_ptr(&self) -> *const c_bindings::utc_tm {
+        &self.0
+    }
+
+    /// Creates a UTC time from its individual components
+    pub fn from_date(year: u16, month: u8, day: u8, hour: u8, minute: u8, second: f64) -> UtcTime {
+        UtcTime(unsafe {
+            c_bindings::date2utc(
+                year as i32,
+                month as i32,
+                day as i32,
+                hour as i32,
+                minute as i32,
+                second,
+            )
+        })
+    }
+
+    /// Number of years CE. In four digit format
+    pub fn get_year(&self) -> u16 {
+        self.0.year
+    }
+
+    /// Day of the year (1 - 366)
+    pub fn get_day_of_year(&self) -> u16 {
+        self.0.year_day
+    }
+
+    /// Month of the year (1 - 12). 1 = January, 12 = December
+    pub fn get_month(&self) -> u8 {
+        self.0.month
+    }
+
+    /// Day of the month (1 - 31)
+    pub fn get_day_of_month(&self) -> u8 {
+        self.0.month_day
+    }
+
+    /// Day of the week (1 - 7). 1 = Monday, 7 = Sunday
+    pub fn get_day_of_week(&self) -> u8 {
+        self.0.week_day
+    }
+
+    /// Hour of the day (0 - 23)
+    pub fn get_hour(&self) -> u8 {
```

Comment:
A lot of getters in this crate have a `get_` prefix but some don't, it would be nice to have it all consistent. FWIW the api guidelines say no prefix - https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter. 

Might be better off in a separate pr, but it would be nice to have it follow the style guide before a release to crates.io. libsbp also uses `get_*` a lot as well so it could use the same treatment

Reply:
I've updated the functions added in this PR to match that style. I'll open a follow on PR to update the existing function names.